src/daemon/control.rs

```diff
@@ -135,7 +122,7 @@ fn bitcoind_wallet_tx(
 
 // Tell bitcoind to broadcast the Unvault transactions of all these vaults.
 // The vaults must be active for the Unvault to be finalizable.
-fn bitcoind_broadcast_unvaults(
+pub fn bitcoind_broadcast_unvaults(
```

Comment:
nit: can you for the functions you make public:
1. Make the existing comment doc comments
2. Make sure they are still valid
3. If there is no comment, add a *meaningful* (not necessarily long) doc comment for it?